utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/SurfaceDuoUtils.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.testing
+
+import android.graphics.Rect
+import android.view.Surface
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+
+/**
+ * Surface Duo 1 dimensions as they can be seen on:
+ * https://docs.microsoft.com/dual-screen/android/surface-duo-dimensions
+ */
+object SurfaceDuo1_dimens {
+    val START_SCREEN_RECT = Rect(0, 0, 1350, 1800)
+    val END_SCREEN_RECT = Rect(1434, 0, 2784, 1800)
+    val SINGLE_SCREEN_WINDOW_RECT = START_SCREEN_RECT
+    val DUAL_SCREEN_WINDOW_RECT = Rect(0, 0, 2784, 1800)
+    val SINGLE_SCREEN_HINGE_RECT = Rect()
+    val DUAL_SCREEN_HINGE_RECT = Rect(1350, 0, 1434, 1800)
+    const val SINGLE_SCREEN_WIDTH = 1350
+    const val HINGE_WIDTH = 84
+    const val SCREEN_COUNT = 2
+    const val DUAL_SCREEN_WIDTH = SINGLE_SCREEN_WIDTH * SCREEN_COUNT + HINGE_WIDTH
+}
+
+/**
+ * Surface Duo 2 dimensions as they can be seen on:
+ * https://docs.microsoft.com/dual-screen/android/surface-duo-dimensions
+ */
+object SurfaceDuo2_dimens {
+    val START_SCREEN_RECT = Rect(0, 0, 1344, 1892)
+    val END_SCREEN_RECT = Rect(1410, 0, 2754, 1892)
+    val SINGLE_SCREEN_WINDOW_RECT = START_SCREEN_RECT
+    val DUAL_SCREEN_WINDOW_RECT = Rect(0, 0, 2754, 1892)
+    val SINGLE_SCREEN_HINGE_RECT = Rect()
+    val DUAL_SCREEN_HINGE_RECT = Rect(1344, 0, 1410, 1892)
+    const val SINGLE_SCREEN_WIDTH = 1344
+    const val HINGE_WIDTH = 66
+    const val SCREEN_COUNT = 2
+    const val DUAL_SCREEN_WIDTH = SINGLE_SCREEN_WIDTH * SCREEN_COUNT + HINGE_WIDTH
+}
+
+/**
+ * Utility function that switches the  application from single screen mode to dual screen on Surface Duo devices.
```

Comment:
The values for span and un-span using `device.swipe` are only for SurfaceDuo1, should we mention that somewhere?

Reply:
Fantastic catch! I will mention that! Thanks!

Reply:
Looks great!